packages/use-contractkit/src/connectors/connectors.ts

```diff
@@ -282,8 +296,9 @@ export class WalletConnectConnector implements Connector {
     );
     localStorage.setItem(
       localStorageKeys.lastUsedWalletArguments,
-      JSON.stringify(options)
+      JSON.stringify([options])
```

Comment:
why wrap this in an array?

Reply:
So because we spread the arguments save in `lastUsedWalletArguments` to the connector. Like
`new Connector(network, ...lastUsedWalletArguments)` I've never seen more than one argument saved into localStorage with this but I figured it was fine to leave as an array.